app/assets/javascripts/lib/ckeditor.js

```diff
@@ -7,4 +8,4 @@ $(document).ready(e => {
   for (let el of document.querySelectorAll('.ckeditor')) {
     CKEDITOR.replace(el.id, {toolbar: 'mini'});
```

Comment:
@cgruppioni yeah, this is still used. You're right that this is where the toolbar is configured and, even though we don't have all the plugins, `{toolbar: 'mini'}` still has an effect. You can see that "mini" config here: https://h2o-dev-test.lil.tools/assets/ckeditor/config.js and the npm version still pulls the same config file. But where it says `config.toolbar_mini = [
    { name: 'paragraph', groups: [ 'list', 'indent', 'blocks', 'align', 'bidi' ], items: [ 'NumberedList', 'BulletedList', '-', 'Outdent', 'Indent', '-', 'Blockquote', 'CreateDiv', '-', 'JustifyLeft', 'JustifyCenter', 'JustifyRight', 'JustifyBlock' ] },` it only displays those items if there's a plugin available to render them so in the case of 'JustifyCenter' etc the plugin is missing and those buttons are missing in this branch. It'd be easy enough to create our own build with the plugins we want or to simply copy and paste the current build on production into our repo, I simply avoided doing that because it's nice to use NPM versions for easy tracking and updating if there's no reason to create a local copy.